t/08-autotest.t

```diff
@@ -449,9 +449,14 @@ subtest make_snapshot => sub {
 };
 
 subtest loadtestdir => sub {
-    throws_ok {
-        autotest::loadtestdir('test-snapshot');
-    } qr/does not exist!/, 'test died as expected';
+    use lib 't/data/tests';
```

Comment:
This is something that shouldn't be necessary. autotest should add the necessary include paths itself.
It seems to me `loadtestdir` hasn't been used for many years, not in our code and not in osado.
The way it works produces warnings, and below you are just kind of ignoring the warnings by comparing it to `ARRAY`, which is just a stringification of an array ref.
The warning is
```
loadtest needs a script below t/data/tests - t/data/tests/tests/assert_screen.pm is not 
```
That's because of the way `loadtestdir` passes the tests to `loadtest`.

I think we should rather fix `loadtestdir` instead, or remove it.

Reply:
So, Should we remove `loadtestdir` completely?

Reply:
https://github.com/search?q=loadtestdir+language%3APerl&type=code&l=Perl
maybe not...
The problematic part seems to be that the `glob()` in `loadtestdir` can return a part of the path that is already in CASEDIR, and then `find_script` will complain about `(!-f join('/', $casedir, $script)`.